doc/source/ray-air/api/configs.rst

```diff
@@ -4,20 +4,21 @@ Ray AIR Configurations
 
 .. TODO(ml-team): Add a general configuration guide that covers all of these configs.
 
-.. currentmodule:: ray
-
-
 .. note::
 
     We are changing the import path of the configurations classes from `ray.air` to `ray.train` starting from Ray 2.7, 
     please see the :ref:`Ray Train API reference <ray-train-configs-api>` for the latest APIs.
 
+.. currentmodule:: ray.train
+
 .. autosummary::
 
-    air.RunConfig
```

Comment:
your call, I don't have an opinion. Just thought that as long as we have the migration warning on top, it might make sense to make it look like this is the "old" one

Reply:
Okay I don't have a strong opinion here either. Thanks!